src/wallet/wallet.h

```diff
@@ -527,6 +527,15 @@ class CWalletTx: public CWalletTransactionBase
             listScSent.push_back(output);
         }
     }
+    template <typename T>
```

Comment:
I think you can drop the template here, since `scFees` are define for mbtr only.  With a bit of more effort, we could merge `fillScSent` and `fillScFees` introducing a `getScAmount `method similarly to what we did for CcOuts in `transaction.h`

Reply:
done